lib/resources/dstu2/soarian_care_plan.yaml

```diff
@@ -0,0 +1,5 @@
+---
+name: CarePlan
+field_name_base_url: http://hl7.org/fhir/DSTU2/careplan-definitions.html#CarePlan
```

Comment:
There's an identifier returned in the example, but the terminology for the identifier type isn't mentioned.